backend/de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/v2/project/ProjectRestService.java

```diff
@@ -66,10 +70,14 @@
 	private final ICurrencyDAO currenciesRepo = Services.get(ICurrencyDAO.class);
 
 	private final ProjectRepository projectRepository;
+	private final ProjectService projectService;
 
-	public ProjectRestService(@NonNull final ProjectRepository projectRepository)
+	public ProjectRestService(
+			@NonNull final ProjectRepository projectRepository,
+			@NonNull final ProjectService projectService)
```

Comment:
leave it like this now,
but usualy when u see repo and service one near the other, 
it's a sign that you shall use only the service and let the service expose the needed repo method(s).